src/app/stargazers/stargazers.component.html

```diff
@@ -23,10 +23,14 @@ <h3 class="mt-1 mb-2">Stargazers</h3>
   <div class="panel-body" *ngIf="starGazers?.length > 0">
     <div fxLayout="row wrap">
       <div *ngFor="let stargazer of starGazers">
-        <div fxFlex="90%">
-          <img src="{{ stargazer.avatarUrl }}" width="150" height="150" class="img-responsive stargazer-img" alt="{{ altAvatarImg }}" />
-          <div class="caption">
+        <div fxFlex="90%" class="mx-3 text-center">
+          <img src="{{ stargazer.avatarUrl }}" class="stargazer-img" alt="{{ altAvatarImg }}" />
+          <div style="width:150px;">
```

Comment:
Don't use pixels. :(

Reply:
Ok :( There were already pixels in use here so I thought it might be ok... are we transitioning away from using them?

Reply:
https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide